tests/sdk/nodejs/helm_test.go

```diff
@@ -0,0 +1,186 @@
+// Copyright 2016-2023, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// nolint:copylocks
+// nolint:goconst
+package test
+
+import (
+	"os"
+	"testing"
+
+	. "github.com/onsi/gomega"
```

Comment:
Could we opt for an explicit import rather than a dot import in this context? This would enhance clarity for reviewers by clearly indicating the source of each function. Since we utilize only a handful of exported functions from gomega in this test file, the impact on code length itself is minimal, while readability would improve significantly.

Reply:
I'm pretty strongly against that because the verbosity takes a big hit and I'm obsessed with concision in test code.